geometry_msgs/Point real_coordinate
---
int32 x
int32 y
